### ISSUE TITLE
Linking Dataverse makes user scroll through list of all Dataverse collections in repository 

### ISSUE THREAD

**Issue by kmika11** [IQSS/dataverse#9092]
**What steps does it take to reproduce the issue?** When linking a Dataverse collection, the "link Dataverse" button opens up a dropdown menu from which a user must select the Dataverse to link to from what looks like a list of all Dataverse collections in the repository. The option to type in the name of a Dataverse collection to link to doesn't exist. 

* When does this issue occur? When clicking on the "Link Dataverse" button on a Dataverse Collection page. 

* Which page(s) does it occurs on? On Dataverse collection pages, but *not* on datasets. 

* What happens? The popup requires a user to scroll through a list of all Dataverse collections until they find the one they want to link to. You can type to skip to a closer dataset, but it's not obvious. 

* To whom does it occur (all users, curators, superusers)? Superusers, all users

* What did you expect to happen? A free text box that will autofill the Dataverse collection name

**Which version of Dataverse are you using?** 5.12

**Any related open or closed issues to this bug report?** I couldn't find any! 

**Screenshots:**

No matter the issue, screenshots are always welcome.

To add a screenshot, please use one of the following formats and/or methods described here:

* https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests
*
<img width="1295" alt="Screen Shot 2022-10-21 at 12 22 19 PM" src="https://user-images.githubusercontent.com/23460791/197243078-863726fc-7487-4ef7-9bb0-6139a31555b8.png">

**Comment by kmika11** [comment]
Just realized that this may be non-ideal for superusers only. Potentially, this list populates with Dataverse collections that the user has privileges to link into. Regardless, this is still (in my opinion!) unwieldy for a user that has a large number of collections (or subcollections), and the feature should operate the same way on both the Dataset and Dataverse collection pages.

**Comment by sbarbosadataverse** [comment]
This issue needs to be completed first: https://github.com/IQSS/dataverse/issues/6467

**Comment by scolapasta** [comment]
Link Dataverse and Link Datasert (both which show you DV collections) have different UI/UX. We should determine one that we end up using for both (once the backend permisisons work is compete):

See https://github.com/IQSS/dataverse/issues/6772

**Comment by cmbz** [comment]
2024/04/22
- Prioritization team discussed this issue
- Dependencies on need for UX/UI input and decisions
- @siacus, @scolapasta , @ekraffmiller and @jggautier will meet to discuss which UI will be implemented
  - During that meeting, the team will draft a process to handle small scope UI/UX decisions like this, internally. Guidance may include recommendations of how to prioritize in context of backend impact and impact for JSF, SPA, etc.
 - @cmbz will create a label (UX/UI Input Needed) to apply to and track issues of this type

**Comment by sbarbosadataverse** [comment]
I tested this issue today. You can start typing the name of the collection and it appears to be working as expected. If you don't know the name of the collection you want to link, that will be more problematic.

**Comment by cmbz** [comment]
@sbarbosadataverse is your observation a show stopper, or can we close the issue?

**Comment by cmbz** [comment]
2025-01-22: Closing as complete. Please create a new issue for feature enhancements.
